### PR TITLE
Feature/case details forms

### PR DESCRIPTION
**Pull request by JohnRyanB**
### 03/23/22:
What work was done?

- Created Client/Family, Household Information, and Notes case detail view forms, as well as the single note component, for the Case Details Collapsible.
- Styled those forms according to the style guide

### 3/30/22
- Reworked Notes.js based on stakeholder feedback

### 3/31/22
- Removed missed commented out code in presentational components

### Why was this work done? 

- Case Managers need to see a client/family's information in a quick, easy to read location with a logical flow of information from most normally used to least in a dark mode style for reduced visual fatigue. [Trello Card](https://trello.com/c/KYCd0jQ1/17-as-a-case-manager-i-want-to-see-the-guest-information-inside-of-a-case-so-that-i-can-work-the-case-for-the-guest-case-details)

### Type of change
- [x]  New feature (non-breaking change which adds functionality)

### Change Status
- [x]  Complete, tested, ready to review and merge
- [ ]  Complete, but not tested (may need new tests)
- [ ]  Incomplete/work-in-progress, PR is for discussion/feedback

### Checklist
- [x] My code follows the style guidelines of this project
- [x]  I have performed a self-review of my own code
- [ ]  My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x]  My changes generate no new warnings
- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  New and existing unit tests pass locally with my changes
- [x]  There are no merge conflicts

### CODE REVIEW

**Review by Rex-1031 — APPROVED** [review]
Nice work everyone. It's good to see this start to come together.

File: src/components/common/CaseDetailsForms/ClientFamilyInformation.js [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+  other: '',
+  veteran: '',
+  pregnant: '',
+  family_member: '',
+  homeless_history: '',
+  employment: '',
+};
+
+const ClientFamilyInfoForm = () => {
+  const [form] = Form.useForm();
```
Comment (stepbystepdayo) [inline src/components/common/CaseDetailsForms/ClientFamilyInformation.js:36]: Ant Design's form is so nice! I hope you guys were able to use the documentation to use this. It's very nicely written!

File: src/components/common/CaseDetailsForms/ClientFamilyInformation.js [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+  const style = {
+    gridTemplateColumns: 'repeat(2, 1fr)',
+    gap: '5px',
+    gridAutoFlow: 'row dense',
+    // paddingTop: '2rem',
```
Comment (stepbystepdayo) [inline src/components/common/CaseDetailsForms/ClientFamilyInformation.js:42]: should we delete the commented out code?

File: src/components/common/CaseDetailsCollapse/CaseDetailsCollapse.js [hunk 3]
```diff
@@ -29,31 +39,29 @@ const CaseDetailsCollapse = () => {
...
-    <div>
+    <div className="CaseDetailsCollapse">
       {/* change h1 text to read appropriate API data when availible */}
       <h1>Case Details</h1>
       <Collapse defaultActiveKey={['notesPreview']} onChange={callback}>
         <Panel
           header="Client Notes"
           key="notesPreview"
+          style={{ margin: 'auto' }}
+          // style={{ backgroundColor: 'red' }}
```
Comment (stepbystepdayo) [inline src/components/common/CaseDetailsCollapse/CaseDetailsCollapse.js:50]: should we delete the commented out code?

File: src/components/common/CaseDetailsForms/ClientFamilyInformation.js [hunk 4]
```diff
@@ -0,0 +1,315 @@
+import React from 'react';
+import {
+  Form,
+  Select,
+  Input,
+  Button,
+  Radio,
+  DatePicker,
+  // TextArea,
```
Comment (stepbystepdayo) [inline src/components/common/CaseDetailsForms/ClientFamilyInformation.js:9]: should we delete the commented out code?

File: src/components/common/CaseDetailsForms/SingleNote.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  const infoStyle = {
+    margin: '.5rem 0',
+    gridRow: '1',
+    gridColumn: '3 / span 2',
+  };
+
+  const pictureStyle = {
+    gridRow: '2',
+    gridColumn: '1',
+    // marginLeft: '3rem',
```
Comment (stepbystepdayo) [inline src/components/common/CaseDetailsForms/SingleNote.js:43]: should we delete the commented out code?

**Review by stepbystepdayo — COMMENTED** [review]
you guys done so much work on the FE! I am so sorry that I can not review so much but maybe we can remove the commented out code.

File: src/components/common/CaseDetailsForms/SingleNote.js (hunk 5 above)
Comment (JohnRyanB) [reply src/components/common/CaseDetailsForms/SingleNote.js]: good catches, I went ahead and removed the commented out code in each component you pointed out.

**Comment by gsauce7** [comment]
Looking good! Nice to see style guide adherence and clean UI coming together.

**Review by ShariqAli-Dev — APPROVED** [review]
I love the design and form factor you made the notes section in. This is exactly what we need and I'm sure the stakeholder would enjoy it.

**Review by stepbystepdayo — APPROVED** [review]
Thank you for removing the commented out code!

**Review by stkirk — APPROVED** [review]
Pulled down the branch and everything is working on my local. Great work John and Chevy. Especially on the notes section. Now we can finally work on state management and api integration!

**Review by ashtilawat23 — APPROVED** [review]

**Pull request merged**